src/TrustedShops.php

```diff
@@ -173,11 +178,11 @@ public static function wp_footer() {
     </script>
   <?php if ($display_product_stars) { ?>
     <script type="text/javascript" src="//widgets.trustedshops.com/reviews/tsSticker/tsProductStickerSummary.js"></script>
-    <script> 
+    <script>
       var summaryBadge = new productStickerSummary();
       summaryBadge.showSummary({
         'tsId': '<?= $shop_id ?>',
-        'sku': ['<?= $product->get_sku() ?>'],
+        'sku': ['<?= $product_sku ?>'],
```

Comment:
PHP Notice, as `$product_sku` is not defined if the current page is not a product.

Reply:
@sun this block of code will not be executed if the current page is not a product. See lines 151-154 and 179. The same for block starting at line 196.